build/build-checkpoint-image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+BOOTKUBE_ROOT=${BOOTKUBE_ROOT:-$(git rev-parse --show-toplevel)}
+
+DOCKER_REPO=${DOCKER_REPO:-quay.io/coreos/checkpoint}
```

Comment:
Can we pick a less generic name -- given it's the coreos top-level account: maybe something like `pod-checkpointer`